sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -552,6 +554,8 @@ def test_one_hot_encoder_specified_categories_mixed_columns():
     assert np.issubdtype(enc.categories_[1].dtype, np.object_)
 
 
+# TODO(1.4): Remove filterwarning when `sparse` is replaced by `sparse_output`.
+@pytest.mark.filterwarnings("ignore:`sparse` was renamed:FutureWarning:sklearn")
```

Comment:
Why is this filter necessary since you now use ``sparse_output`` in ``check_categorical_onehot`` ?

Reply:
Fixed it :+1: